src/obsolete/O3_alternative.jl

```diff
@@ -0,0 +1,169 @@
+# Alternative to the computation of rotation equivariant coupling coefficients
```

Comment:
If this is to become the new main implementation of O3, can you please move it to the main src folder and rename it `O3.jl`.

Reply:
I guess this can hardly be done right now because O3_alternative can just provide the coupling coefficients for the complex spherical vector case. It is still missing how the CCs for the real spherical vectors can be obtained in this way. So I think we can retain both (or even do a merge?) for now? I will no matter what file an issue mentioning this.